.github/workflows/pr_build_jdk11.yml

```diff
@@ -6,10 +6,43 @@ on:
       - master
 
 jobs:
-  windup-build:
 
+  windup-build-with-unit-tests-1:
     runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2.3.4
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+        java-package: jdk
+        cache: 'maven'
+    - name: Set up chromedriver variable
+      run: |
+        echo "CHROMEWEBDRIVER=${CHROMEWEBDRIVER}" >> "$GITHUB_ENV"
+    - name: Build on JDK 11
+      run: mvn clean install -DskipTests -B -U && mvn -B clean install -pl '!tests,!rules-java/tests,!rules-xml/tests,!config/tests' -Dwebdriver.chrome.driver=${{ env.CHROMEWEBDRIVER }}/chromedriver
```

Comment:
Why do we have to run `install` twice? Wouldn't we save some time by running `test` the second time, or does that not work?

Reply:
As discussed, I've just kept it since I was thinking it was related to some issue in creating this job.
Since it looks like there were no issue, let's switch to have `test` instead.

Reply:
This triggered some failure, e.g. `java.io.FileNotFoundException: Source '/home/runner/work/windup/windup/forks/jdt/../../forks/gremlin-shaded/target/classes' does not exist` when running `org.jboss.windup.graph.duplicate.typevalue.DuplicateTypeValueTest`